### DISCUSSION TITLE
Are Characteristic Indication subscribable?

### DISCUSSION THREAD

**Discussion by ben91187** [PhilipsHue/flutter_reactive_ble#706] [Q&A]
Is it possible to subscribe for Characteristic Indications? 
I just found subscribeToCharacteristic without any option to select between notify or indication.

**Comment by TheAlbertDev** [comment]
Did you get an answer or a solution? I had not noticed that the discussions were activated and I have created an [issue](https://github.com/PhilipsHue/flutter_reactive_ble/issues/823) with this doubt... 😅 (But I see that you asked in March and you didn't get an answer... I don't know if you finally got information about it or not).

I would be very grateful for any information/help in this regard. 🙇‍♂️
